core/client/assets/sass/patterns/forms.scss

```diff
@@ -250,6 +250,11 @@ textarea {
                 opacity: 1;
             }
         }
+
+        input:disabled + .input-toggle-component {
+            background: $midbrown;
+            border-color: darken($midbrown, 10%);
+        }
```

Comment:
@JohnONolan @PaulAdamDavis I took the liberty of making a disabled style for the checkboxes. /me ducks :bird: